week2/project/ecommerce/src/components/pages/ProductDetailsPage.jsx

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import { useCallback, useEffect, useState } from "react";
+import ProductCard from "../ProductCard";
+import { useParams } from "react-router-dom";
+
+function ProductDetailsPage() {
+  const { productId } = useParams();
+
+  const [product, setProduct] = useState();
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const getProductDetails = useCallback(async () => {
+    await axios(`https://fakestoreapi.com/products/${productId}`)
+      .then((responce) => {
```

Comment:
it's `response` I usually use `Code Spell Checker` extension to warn me if I did a spelling mistake
📝advise📝